docs/deployment/central-backup-server.rst

```diff
@@ -124,9 +124,7 @@ folder. Even when the configuration is changed the repository server configurati
 satisfied and reproducible.
 
 Automate setting up an repository server with the user, group, folders and
-permissions a Ansible playbook could be used. Keep in mind the playbook
-uses the Arch Linux `pacman <https://www.archlinux.org/pacman/pacman.8.html>`_
-package manager to install and keep borg up-to-date.
+permissions a Ansible playbook could be used.
```

Comment:
not related to your change, but while we're at it:

parse error. can you fix so this to understandable, correct english?

Reply:
Is maybe this what is meant?:

**To** automate setting up **a** borg repository server **an** Ansible playbook could be used.